events/wc_offmap_events.txt

```diff
@@ -786,4 +786,29 @@ narrative_event = {
 	option = {
 		name = EVTOPTA_WCOME.201
 	}
+}
+character_event = {
+	id = WCOME.250
+	picture = GFX_evt_cthun
+	desc = EVTDESC_WCOME.250
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+	
+	trigger = {
+		NOT = { religion = old_gods_worship }
+		owns = 2956j
+		b_ahnqiraj = {
+			has_building = ca_old_god_1
+		}
+	}
+	
+	option = {
+		name = EVTOPTA_WCOME.250
+		e_herald_nzoth = {
```

Comment:
e_herald_nzoth gets c_ahnqiraj. Is it ok?

Reply:
fixed